docs/source/workflow/generate-types.mdx

```diff
@@ -216,4 +201,5 @@ Check out our example using Apollo Server with generated types on CodeSandbox:
     src="https://codesandbox.io/static/img/play-codesandbox.svg"
   />
 </a>
+<br>
```

Comment:
Should this be here?

And TIL bare `<br>`'s (not self-closing) are ok
https://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br

Reply:
This creates extra space between the CodeSandbox button and the "Edit on GitHub" button for smaller screens. So it's not necessary at all; I just think it looks neater 😅